### PR TITLE
feat(CBH): add cbh flavors instance

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  add cbh flavors instance
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  add cbh flavors instance
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run=TestAccSomethingV0_basic'
...
make testacc TEST='./huaweicloud/services/acceptance/cbh/' TESTARGS='-run TestAccDatasourceCbhFlavors_basic' 
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/cbh/ -v -run TestAccDatasourceCbhFlavors_basic -timeout 360m -parallel 4 
=== RUN   TestAccDatasourceCbhFlavors_basic 
=== PAUSE TestAccDatasourceCbhFlavors_basic
=== CONT  TestAccDatasourceCbhFlavors_basic
--- PASS: TestAccDatasourceCbhFlavors_basic (6.45s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cbh       6.501s
```

### CODE REVIEW

File: huaweicloud/services/acceptance/cbh/data_source_huaweicloud_cbh_flavors_test.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  }
+  flavors {
+    resource_spec    = "cbh.basic.50"
+    region           = "cn-north-4"
+    period_unit      = "month"
+    period           = 1
+    subscription_num = 1
+  }
+}
+`, acceptance.HW_PROJECT_ID)
```
Comment (deer-hang) [inline huaweicloud/services/acceptance/cbh/data_source_huaweicloud_cbh_flavors_test.go:78]: If `acceptance.HW_PROJECT_ID` is required. Please add it in `PreCheck`

File: docs/resources/cbh_flavors.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+Use this data source to get the list of CBH flavor.
+
+## Example Usage
+
+```hcl
+variable "project_id" {}
+
+variable "cloud_service_type" {}
+variable "flavor_1" {}
+variable "flavor_2" {}
```
Comment (deer-hang) [inline docs/resources/cbh_flavors.md]: please add the detaile variable field

Comment (houpeng80) [reply docs/resources/cbh_flavors.md]: done

File: huaweicloud/services/acceptance/cbh/data_source_huaweicloud_cbh_flavors_test.go (hunk 1 above)
Comment (houpeng80) [reply huaweicloud/services/acceptance/cbh/data_source_huaweicloud_cbh_flavors_test.go:78]: done

**Pull request closed without merging**